doc/_templates/pandas_footer.html

```diff
@@ -1,3 +1,3 @@
 <p class="copyright">
-    &copy {{copyright}} pandas via <a href="https://numfocus.org">NumFOCUS, Inc.</a> Hosted by <a href="https://www.ovhcloud.com">OVHcloud</a>.
```

Comment:
Is this still needed if we use the comma ending?

Reply:
It's not _strictly_ needed _either_ way, but not doing it means you get 2 spaces or comma-then-space where there was previously one space

Reply:
The comma-then-space format seems OK to me